runc.go

```diff
@@ -765,15 +766,23 @@ func (r *Runc) Features(context context.Context) (*features.Features, error) {
 	return &feat, nil
 }
 
-func (r *Runc) args() (out []string) {
+func (r *Runc) args(logPath string) (out []string) {
 	if r.Root != "" {
 		out = append(out, "--root", r.Root)
 	}
 	if r.Debug {
 		out = append(out, "--debug")
 	}
 	if r.Log != "" {
-		out = append(out, "--log", r.Log)
+		if logPath != "" {
```

Comment:
you don't want this case here
```
if r.Log != "" {
 		if logPath != "" {
                    out = append(out, "--log", logPath)
```
 because you have it below.. Maybe:
```
	if r.Log != "" {
 		if logPath == "" {
 			out = append(out, "--log", r.Log)
 		}
 	} 
        if logPath != "" {
 		out = append(out, "--log", logPath)
        }
```

just in case let's do a unit test for args, pls set up a truth table..

Reply:
done, thank you